Docs/python_api.md

```diff
@@ -710,49 +668,6 @@ Converts the image to a depth map using a logarithmic scale, leading to better p
 No changes applied to the image. Used by the [RGB camera](ref_sensors.md#rgb-camera).  
 
 ---
-## carla.CustomV2XData<a name="carla.CustomV2XData"></a>
```

Comment:
It is contribution of the community I think we shouldn't remove it

Reply:
I think that the user may have added the docs directly to python_api.md rather than to the .yml files in PythonAPI/docs. Now that I made the changes Jorge suggested for get_required_files, the autogeneration of python_api.md has overwritten this user contribution. I will migrate the user docs to the yml files before merging this